src/Runtime/Eval.hs

```diff
@@ -56,21 +52,14 @@ bindings vs = map (bind x) vs
     x = map (bind []) vs
 
 -- | Bind an individual valdef to its name in the current Enviroment
```

Comment:
Should the description be changed here to reflect call-by-value, or is this still accurate from before?

Reply:
Yeah, fixed